src/pages/CropSidebar/PreviousCashCrop/PreviousCashCrop.js

```diff
@@ -21,7 +21,7 @@ const PreviousCashCrop = () => {
       <Grid
         item
         xs={12}
-        style={{ paddingTop: '2rem' }}
+        style={{ paddingTop: '2rem', paddingBottom: '7px' }}
```

Comment:
ted made a good suggestion to not add padding or margin and margin to Grids

Reply:
you can put padding on a box that contains a grid though

Reply:
ahh didnt think about that i updated it.